tests/State/LayerTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of Raven.
```

Comment:
Not really specific to this file, but we should, in the end, decide what to do with this `License` headers. I don't really find the necessary and helpful.
Also we do not want to use `Raven` as a term anywhere ^^

Reply:
Agree on this, but since that's enforced by the code style fixer, it should be done in a separate PR, since it will generate a huge diff and build failures on any new file from other PRs...

Reply:
At the moment I would like to add it to all files because this is what we have done until now. Before finalizing `2.0` we could simply change all the references of `Raven` to `Sentry` or simply drop the license headers all together as part of a big CS cleanup. WDYT?

Said that, I think that the license header is fine as is not too big and helps understanding that the source code is provided under a license. However I'm not a layer, so I don't know if it's realyl necessary to have it or the LICENSE file in the root of the project is enough. Looking at the other projects there doesn't seems to be a standard, everyone does what he prefers

Reply:
I am +1 one for just removing them, doesn't really add any value in my eyes. We have a License and Contributors file in the Repo. I will add it to the todo list before we release 2.0.